### PR TITLE
Add nostr

### PR DESCRIPTION
**Pull request by wyatthilde**
- Add Nostr functionality
- use jsx
- implement react bulma components
- refactored/simplified some things

### CODE REVIEW

File: client/src/HagahStore.jsx
Comment (wyatthilde) [inline client/src/HagahStore.jsx:1]: This is in prep for adding a front-end search on the nostr content

File: client/src/pages/HomePage.jsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import { Link, useLocation } from "react-router-dom";
+import Header from "../components/Header";
+
+function HomePage() {
+  return (
+    <div>
+      <div className={styles.homePageContainer}>
+      <div className={styles.sections}>
+        <h4 className="title is-4 subtitle">Sections</h4>
+
```
Comment (Theobasallaje) [inline client/src/pages/HomePage.jsx]: Please look at this commit and make sure it gets merged in
https://github.com/IJNA/the-orb/commit/45792281306f6a9a2cfe04fdd59bf91a863c1277
Thanks!

Comment (Theobasallaje) [reply client/src/pages/HomePage.jsx]: @wyatthilde 
I made another minor UI update

Here is that commit: https://github.com/IJNA/the-orb/commit/25b0a80093f02d31da7ef0155b99733176fca370

Thanks!

**Review by Theobasallaje — APPROVED** [review]
@wyatthilde Looks good, thanks!

**Pull request merged**